src/components/app/App.js

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
+import Routes from '../../routes';
 import './App.css';
 
-import PageSideNav from './sidenav/PageSideNav';
-import PageContent from './content/PageContent';
-
 const App = () => {
   return (
     <div className="App fill-container" >
```

Comment:
A classe `App` não existe mais

```suggestion
     <div className="fill-container" >
```